content/TrueCommand/Administration/Settings/SAML.md

```diff
@@ -42,149 +40,148 @@ To configure AD as the IdP, you must create and modify a Relying Party Trust. Af
+   e. Locate the <file>tc.cer</file> file (**C:/local disk/users/QE** and then select the <file>tc.cer</file>. Click **Open** to view the **Configure Certificate** window and see information about the certificate. Click **Next** to move on to the **Configure URL** window.
 
-4. Configure the URL. On the Configure URL window:
+4. Configure the URL. In the **Configure URL** window:
 
-   a. Select Enable support for the SAML 2.0 webSSO protocol. 
+   a. Select **Enable support for the SAML 2.0 WebSSO protocol**. 
         
    ![WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped](/images/SAML/WizardSelectEnableSupportForSAML2_0WebSSOProtocolCropped.png "Wizard Select Enable Support For SAML 2.0 Web SSO Protocol Cropped")
         
-   b. Type or copy/paste the TrueCommand login URL http://IP:PORT/saml/acs in the Relying party SAML 2.0 SSO service URL field. Where IP:PORT is your TrueCommand system IP and port address.
+   b. Type or copy/paste the TrueCommand login URL (*http://IP:PORT/saml/acs*) in the **Relying party SAML 2.0 SSO service URL** field. *IP:PORT* is your TrueCommand system IP and port address.
 
-   c. Click **Next**. The Configure Identifiers window displays.
+   c. Click **Next** to continue to the **Configure Identifiers** window.
 
 5. Configure the SAML identifiers.
 
-   a. Type truecommand-sml into the Relying party trust identifier field and click Add.
+   a. Type *truecommand-sml* into the **Relying party trust identifier** field and click **Add**.
 
    ![WizardConfigureIdentifiersTypeNameCropped](/images/SAML/WizardConfigureIdentifiersTypeNameCropped.png "Wizard Configure Identifiers Type Name Cropped")
         
-   b. Click **Next**. The Choose Access Control Party window displays. 
+   b. Click **Next** to move on to the **Choose Access Control Party** window. 
 
-   c. Click **Next**. The Ready to Add Trust window displays. Click **Next** and then click **Finish**. 
+   c. Click **Next** again to view the **Ready to Add Trust** window, then click **Next** once more and select **Close**. 
 
    ![WizardClickFinishCropped](/images/SAML/WizardClickFinishCropped.png "Wizard Click Finish Cropped")
 
 6. Modify the newly-created Relying Party Trust.
 
-   a. Select the new SAML Relying Party Trust and then select Properties on the Actions menu displayed on the right side of the screen. The Properties window displays. 
+   a. Select the new SAML Relying Party Trust and then select **Properties** in the **Actions** menu to open the **Properties** window. 
 
    ![SelectRelyingPartyTrustAndProperties](/images/SAML/SelectRelyingPartyTrustAndProperties.png "Select Relying Party Trust And Properties")
         
-   b. Select the Endpoints tab and click **Edit**. The Edit Endpoint window displays.
+   b. Select the **Endpoints** tab, then click **Edit** to open the **Edit Endpoint** window.
 
-   c. Change the Index value to 1 and click **OK**. 
+   c. Change the **Index** value to *1* and click **OK**. 
```

Comment:
1 is the fixed value not a variable value, suggest changing it to **1**

   c. Change the **Index** value to **1** and click **OK**.

Reply:
Fixed